build/storage/tests/it/ptf_tests/tests/test_hot_plug.py

```diff
@@ -64,6 +64,7 @@ def runTest(self):
         )
 
     def tearDown(self):
+        self.platforms_factory.get_cmd_sender().stop()
```

Comment:
I still have concerns if we need cmd sender at all and we need to send cmds directly from tests. Based on that, can we remove cmd_sender container stopping out of the test code?